testdata/license_header_template.txt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 The Trickster Authors
+ * Copyright 2025 The Trickster Authors
```

Comment:
This needs to remain 2018 since that was the year we donated the project to open source. For OSS projects, the copyright year in the license header indicates the year the work was copyrighted, and not when the file was last modified.

Reply:
got it

Reply:
ok, resolved